packages/lpa-questionnaire-web-app/src/views/layouts/main.njk

```diff
@@ -0,0 +1,94 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+
+{% block head %}
+  {% include "includes/head.njk" %}
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "/",
+    serviceName: '',
+    serviceUrl: "#",
+    containerClasses: "govuk-width-container"
+  }) }}
+{% endblock %}
+
+{% set mainClasses = mainClasses | default("govuk-main-wrapper--auto-spacing") %}
+
+{% block main %}
+  <div class="govuk-width-container govuk-body {{ containerClasses }}">
+
+    <main id="main-content" role="main"{% if mainLang %} lang="{{ mainLang }}"{% endif %}>
+      {{ govukPhaseBanner({
+        tag: {
+          text: "beta"
+        },
+        html: 'This is a beta service – your <a class="govuk-link" data-cy="Feedback" href="https://www.smartsurvey.co.uk/s/7OGXZZ/">feedback</a> will help us to improve it.'
+      }) }}
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "javascript:history.back()",

```

Comment:
there's debate around this as well, not sure what but this might not be right.

Reply:
As above, need some more info and need to be kept up to date on this

Reply:
Jamie Crisp (from PINS) is the person to ask about this, he is the UX lead for these projects. 

I don't know what it is they want here, but they have mentioned we copied this from their prototype and it's not the expected behaviour. I've not seen a ticket for what is the expected behaviour, but I believe this will need to change.

Pretty much same issue as GA. I'm not working on it, so I might not have immediate visibility when the requirement changes, it's more of a heads up.

I'll raise both of these issues internally so the entire team is aware.

Reply:
OK thanks, I believe we're talking to Jamie around the tickets ourselves so will bring that up with him.